api/elasticdljob/v1alpha1/elasticdljob_types.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 The Kubeflow Authors
```

Comment:
It can be "Alibaba authors", see `./hack/boilerplate.go.txt` for details

Reply:
Done